pkg/kademlia/config.go

```diff
@@ -29,6 +29,8 @@ type Config struct {
 
 	// TODO: reduce the number of flags here
 	Alpha int `help:"alpha is a system wide concurrency parameter" default:"5"`
+	// TODO: make this a shorter amount of time
+	QueryTimeout int32 `help:"query timeout is the allotted time in seconds for a query to complete" default:"60"`
```

Comment:
Nit: We could maybe even reduce this more aggressively, and if the user needs to tweak it they can. But honestly 60s is fine given how long we know these can take.

Reply:
I think this should be `time.Duration`. It will be more generic.